learn/classes_objects.md

```diff
@@ -0,0 +1,219 @@
+Python is a powerful programming language that supports object-oriented programming through the use of classes and objects. With Python class methods, developers can create reusable code structures that help organize and simplify their programs. To make use of these classes in Python, developers can import the necessary class and use the available class methods to create instances and modify properties. In this article, we'll explore an example of how to use Python classes and objects to build robust programs.  
+  
+## Introduction to Python classes: understanding the concept of objects in Python  
+
+In Python, **classes** are a way to group related data and functions into a reusable structure. An **object** is an instance of a class, with its own specific values for the data in the class.
+
+To create a class in Python, use the `class` keyword and define the class methods inside of it. For example:
+
+```python
+class Car:
+    def __init__(self, color, brand):
+        self.color = color
+        self.brand = brand
+    
+    def info(self):
+        print("I am a {} {} car".format(self.color, self.brand))
+
+my_car = Car("red", "Toyota")
+my_car.info()
+```
+
+This creates a `Car` class with an `__init__` method that takes a color and brand as parameters, and an `info` method that prints out information about the car.
+
+To create an **object** of the `Car` class, use the class name followed by parentheses and any necessary arguments:
```

Comment:
The `:` in the end of sentence means the subsequent list. But there's no list after that.